packages/vite/src/node/plugins/css.ts

```diff
@@ -778,6 +779,30 @@ async function compileCSS(
   }
 }
 
+export function createHTMLCSSCompiler(config: ResolvedConfig) {
```

Comment:
Due to this function is `export`ed, could we add a return typedef to strength the TS code?
This prevents accidentally unintended code changes in the future within the function body

Reply:
oh this is good idea, thanks.